libvast/builtins/operators/serve.cpp

```diff
@@ -285,7 +290,9 @@ struct managed_serve_operator {
     }
     // Cut the results buffer.
     auto results = std::vector<table_slice>{};
+    VAST_WARN("getting {}/{} events", requested, rows(buffer));
```

Comment:
Should this really be warning? (Applies to comment below as well.)

Reply:
Ah, the classic debugging left-over